components/ApplicationsList/ApplicationCard.js

```diff
@@ -36,12 +36,20 @@ const ApplicationCard = ({ application }) => {
           textAlign={"center"}
           justifyContent={"center"}
         >
-          <Avatar
-            size="xl"
-            src={application.mentor.picture + "?tr=w-100,h-100,fo-auto"}
-            name={application.mentor.name}
-            alignSelf={"center"}
-          />
+          {application.mentor.picture ? (
+              <Avatar
+                  size="xl"
+                  src={application.mentee.picture + "?tr=w-100,h-100,fo-auto"}
+                  name={application.mentee.name}
+                  alignSelf={"center"}
+              />
+          ) : (
+              <Avatar
+                  size="xl"
+                  name={application.mentee.name}
+                  alignSelf={"center"}
+              />
+          )}
```

Comment:
Congratulations on your first PR!

The two `true`/`false` branches have a lot in common, maybe we could simplify to just have an empty `src` if not `application.mentor.picture`?

The network request should not be triggered in that case.

Removing the `src` if not needed also could be a viable solution.

Reply:
also, is it `application.mentor` or `application.mentee` here?

Reply:
Ouch, being caught copy/pasting on the first PR...

> The two true/false branches have a lot in common, maybe we could simplify to just have an empty src if not application.mentor.picture?

I had a look at the Avatar doc from Chakra and I did not see the empty `src` has a solution, but maybe it is an obvious solution, I will try and see how it rect with empty `src`

Reply:
> I had a look at the Avatar doc from Chakra and I did not see the empty `src` has a solution, but maybe it is an obvious solution, I will try and see how it rect with empty `src`

You can play around in the [doc page for the Avatar](https://chakra-ui.com/docs/components/avatar) which is editable, and you could see using the DevTools that there is no network request when the `src` is empty. To be tested in this situation, maybe the editable box behaves differently.

Reply:
> Ouch, being caught copy/pasting on the first PR...

No worries ^^ request review again when you're ready ;)